clients/frontend-authorization/src/oidc/browser/BrowserAuthorizationClient.ts

```diff
@@ -42,7 +43,8 @@ export interface BrowserAuthorizationClientConfiguration {
 }
 
 /**
- * @beta
+ * Utility used for authorization for Single Page Applications running in a browser
+ * @public
  */
 export class BrowserAuthorizationClient extends BrowserAuthorizationBase<BrowserAuthorizationClientConfiguration> implements FrontendAuthorizationClient, IDisposable {
   public readonly onUserStateChanged = new BeEvent<(token?: AccessToken) => void>();
```

Comment:
Before making this public I would like to try and remove the dependency on the `AccessToken` class for the new  `AccessTokenProps` interface.  This would help with the tight coupling between the two client libraries.

This may be a big change though... I'm prototyping it now to see how extensive it would be.

Reply:
Not sure I follow - can you explain what the proposal is? Note that even if this is marked beta, it's used quite a bit. So we need to be a little careful on changing this.